### ISSUE TITLE
Magento 2.3.0 logo not uploaded due to deferred.pipe() console error

### ISSUE THREAD

**Issue by pratikkamani** [magento/magento2#19681]
Please check attachment. I have fresh Magento 2.3.0 installation. Unable to upload logo.
Error says "A technical problem with the server created an error. Try again to continue what you were doing. If the problem persists, try again later."
Consol error
JQMIGRATE: deferred.pipe() is deprecated    at   /adminhtml/Magento/backend/en_US/jquery/jquery-migrate.jsjquery-migrate.js:45 

Actual result (*)

1. [Screenshots, logs or description]

![untitled](https://user-images.githubusercontent.com/9292282/49713655-af743c80-fc6f-11e8-8646-e9abbe2168cd.png)

**Comment by magento-engcom-team** [comment]
Hi @pratikkamani. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@pratikkamani do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by kunj1988** [comment]
I don't know why but it is working for me.

**Comment by kishanpatadia** [comment]
@magento-engcom-team  I have the same issue in my localhost.
Magento: 2.3.0
Operating System: Ubuntu 14.04
PHP: 7.2.13
Apache: 2.4.33

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
HI @pratikkamani, thank you for you report, i'm not able to reproduce following steps you described, on clean magento 2.3.0 instance.

**Comment by cotiga** [comment]
### Preconditions
same problem on 2.3.0 clean install
Ubuntu 18.04
Nginx 1.14
PHP 7.2.13
Mysql 5.7
### Steps to reproduce
In Content > Configuration > Edit (Luma)
### Result
Not possible to upload favicon, logo...
JQMIGRATE: deferred.pipe() is deprecated

**Comment by slackerzz** [comment]
same here on a clean 2.3.0 enterprise
